### PR TITLE
Additional SQL challenge

### PR DESCRIPTION
**Pull request by c-martinez**
When we ran this lesson, we had a bit of spare time towards the end of the lesson, so we set the learners a number of *challenges* to explore the data a bit further. This worked quite well to keep learners engaged, and to have a feel of how they could use SQL to answer questions on their own.

### CODE REVIEW

File: 03-sql-joins-aliases.md [hunk 1]
```diff
@@ -83,5 +83,18 @@ The `AS` isn't technically required, so you could do
...
+> SQL queries help us *ask* specific *questions* which we want to answer about our data. The real skill with SQL is to know how to translate our scientific questions into a sensible SQL query (and subsequently visualize and interpret our results).
+>
+> Have a look at the following questions; these questions are written in plain English. Can you translate them to *SQL queries* and give a suitable answer?
+> - How many plots from each type are there?
+> - How many specimens are of each gender are there for each year?
+> - How many specimens of each species were captured in each type of plot?
+> - What is the average weight of each taxa?
+> - What is the percentage of specimens in each taxa?
+> - What are the minimum, maximum and average weight for each species of Rodent?
+> - What is the hindfoot_length for male and female rodent of each species? Is there a Male / Female difference?
```
Comment (ethanwhite) [inline 03-sql-joins-aliases.md]: I'd probably remove the `_` in `hindfoot_length` since these are supposed to be plain English questions.

File: 03-sql-joins-aliases.md [hunk 2]
```diff
@@ -83,5 +83,18 @@ The `AS` isn't technically required, so you could do
...
+> - What is the average weight of each rodent species over the course of the years? Is there any noticeable trend on the species?
```
Comment (ethanwhite) [inline 03-sql-joins-aliases.md]: "Is there any noticeable trend on the species?" -> "Is there any noticeable trend for any of the species?"

**Comment by ethanwhite** [comment]
Thanks for the contribution @c-martinez! I like the idea of some extra, plain language, challenge questions at the end in case there is extra time. I think it would be good to indicate that these are optional somehow, maybe just by adding an `(optional)` either before or after `Challenge`. I also suggested a couple of minor changes in in-line comments.

**Comment by c-martinez** [comment]
Hi @ethanwhite , thanks for your suggestions. I've incorporated them and added to this PR.

**Comment by ethanwhite** [comment]
Looks great. Thanks!
